File: app/src/main/java/org/mifos/selfserviceapp/models/ChargeListResponse.java
package org.mifos.selfserviceapp.models;

import java.util.ArrayList;
import java.util.List;

/**
 * @author Vishwajeet
 * @since 16/8/16.
 */
public class ChargeListResponse {
    private List<Charge> pageItems = new ArrayList<>();

    public List<Charge> getPageItems() {
        return pageItems;
    }

    public void setPageItems(List<Charge> pageItems) {
        this.pageItems = pageItems;
    }
}

File: core/datastore/src/main/java/org/mifos/mobile/core/datastore/model/ChargeListResponse.kt
package org.mifos.mobile.core.datastore.model

import org.mifos.mobile.core.datastore.model.Charge

/**
 * @author Vishwajeet
 * @since 16/8/16.
 */
data class ChargeListResponse(
    var pageItems: List<Charge> = ArrayList(),
)
